README.md

```diff
@@ -195,25 +158,9 @@ If you create a feature request, use the dedicated **Feature request** issue tem
 
 If you have a bug to report, reproduce it in an online code editor. For example, use [CodeSandbox](https://codesandbox.io/). Attach the link to the reproduced bug to your issue. Log the bug using the **Bug report** template.
 
-## Missing features
-
-See the list of features that are still missing in the component:
-
-- [ ] render [`bindings`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#fixed-fields-19)
-- [ ] render [`tags`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#tagsObject)
-- [ ] render [`externalDocs`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#externalDocumentationObject)
-- [ ] render [`schema properties`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#properties) 
-- [ ] render [`correlationID`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#correlationIdObject)
-
-If you want to help us develop them, feel free to contribute.
-
 ## Credits
 
-<p align="center">
- <a href="https://kyma-project.io/" target="_blank">
-  <img src="https://raw.githubusercontent.com/kyma-project/kyma/master/logo.png" width="235">
- </a>
-</p>
+The project was originally developed under the [Kyma project](https://kyma-project.io/), in 2020 it was transferred to the AsyncAPI Initiative.
```

Comment:
2019 😉 
I know, time flies so fast, right? 😄

Reply:
I wrote year when Kyma give AsyncAPI 100% control of project :) I don't know if we need to clarify it.

Reply:
you write transferred, transferred/moved/handed over, whatever word you choose, the right date is 2019. Unless you write `in 2020 it was handed over under maintenance of the AsyncAPI Initiative` or `in 2019 it was moved under AsyncAPI Initiative`. Still think the 2nd one is better, but I care more about wording

Reply:
I changed to `in 2019 it was moved under AsyncAPI Initiative` :) Thanks!